WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -3600,7 +3600,7 @@ function WeakAuras.PositionOptions(id, data, _, hideWidthHeight, disableSelfPoin
   return positionOptions;
 end
 
-function WeakAuras.BorderOptions(id, data, _, showBackDropOptions)
+function WeakAuras.BorderOptions(id, data, _--[[???]], showBackDropOptions)
```

Comment:
u wot m8

Reply:
hmm?

Reply:
what is that comment

Reply:
It's a reaction to the fact that we have an underscore in a function instead of just removing the parameter.

Reply:
Wanna remove it then or make it a proper TODO comment?